doc/z2csl.svg

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Wouldn't this be obvious from the XML file itself? Not sure that the diagram really clarifies this.